sql/src/test/java/io/crate/integrationtests/CircuitBreakerIntegrationTest.java

```diff
@@ -62,4 +66,18 @@ public void testQueryBreakerIsDecrementedWhenQueryCompletes() {
 
         assertThat(queryBreaker.getUsed(), is(breakerBytesUsedBeforeQuery));
     }
+
+    @Test
+    public void testQueryBreakerIsUpdatedWhenSettingIsChanged() {
+        execute("create table t1 (text string) clustered into 1 shards");
+        ensureYellow();
+        execute("insert into t1 values ('this is some text'), ('other text')");
+        refresh();
+
```

Comment:
maybe we should validate that the initial setting is not `100b` by issuing the same `select ...` statement before adjusting the setting to really ensure that then change worked? (could be that now or then something changes and the initial setting could be `100b` by coincidence...